pkg/utils/filesystem/fakefs.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/spf13/afero"
 )
 
-// fakeFs is implemented in terms of afero
-type fakeFs struct {
+// FakeFs is implemented in terms of afero
+type FakeFs struct {
```

Comment:
This was needed so we can validate in unit tests that indeed `Fs` was replaced by FakeFs to avoid corruption of the test system